src/gui_elements.py

```diff
@@ -1,16 +1,17 @@
 import os
-
 import pygame
 
 from typing import Callable, Any
+from pygame.locals import Rect
```

Comment:
Imports should be grouped in the following order:

Standard library imports.
Related third party imports.
Local application/library specific imports.